### DISCUSSION TITLE
Include BMW Motorrad Connected App

### DISCUSSION THREAD

**Discussion by MidnightRacer99** [bimmerconnected/bimmer_connected#324] [Feature request]
Hi, Great job you are doing with this.

I'm a motorcycle owner and think it would be great if you can update the compatibility to include the BMW Motorrad Connected App so we can get the information for our motorcycles too.

Many thanks and keep up the good work!
Kev

**Comment by rikroe** [comment]
The BMW Motorrad app currently seems to be completely disconnected from the APIs used for BMW and Minis. If there is a larger upgrade to the app, we should revisit it though!
